### PR TITLE
added node js sample for tab-external auth

### PR DESCRIPTION
**Pull request by Prithvi-MSFT**

### CODE REVIEW

File: samples/tab-external-auth/nodejs/Images/oauthapp2.png
Comment (Pawank-MSFT) [inline samples/tab-external-auth/nodejs/Images/oauthapp2.png:1]: Need to hide ClientID, Secret and url.

File: samples/tab-external-auth/nodejs/Images/redirect-page.png
Comment (Pawank-MSFT) [inline samples/tab-external-auth/nodejs/Images/redirect-page.png:1]: hide ngrok url from Image if possible

File: samples/tab-external-auth/nodejs/api-server/server.js [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+    var googleAppId = process.env.GoogleAppId;
+    var googleAppPassword = process.env.GoogleAppPassword;
+    var redirectUrl = `${process.env.ApplicationBaseUrl}/auth-end`;
+    var idToken = req.query.idToken;
+
+    var url = `https://oauth2.googleapis.com/token?client_id=${googleAppId}&client_secret=${googleAppPassword}&code=${idToken}&redirect_uri=${redirectUrl}&grant_type=authorization_code`;
+    axios.post(url, postData)
+        .then(async response => {
+            var url = `https://people.googleapis.com/v1/people/me?personFields=names,emailAddresses,photos,urls`;
```
Comment (Pawank-MSFT) [inline samples/tab-external-auth/nodejs/api-server/server.js:35]: base URL set in env file.

**Review by Pawank-MSFT — COMMENTED** [review]
please fix given comments

File: samples/tab-external-auth/nodejs/Images/oauthapp2.png
Comment (Prithvi-MSFT) [reply samples/tab-external-auth/nodejs/Images/oauthapp2.png:1]: changed image

File: samples/tab-external-auth/nodejs/Images/redirect-page.png
Comment (Prithvi-MSFT) [reply samples/tab-external-auth/nodejs/Images/redirect-page.png:1]: changed image

**Review by Pawank-MSFT — APPROVED** [review]
looks correct, Approving!

**Comment by Pawank-MSFT** [comment]
Executed build pipeline: 
https://github.com/OfficeDev/Microsoft-Teams-Samples/actions/runs/6734050934

**Review by Wajeed-msft — APPROVED** [review]

**Comment by Pawank-MSFT** [comment]
Executed Build pipeline:
https://github.com/OfficeDev/Microsoft-Teams-Samples/actions/runs/6767459581

**Pull request merged**